app/views/tmdb/_director_credits.html.erb

```diff
@@ -1,13 +1,13 @@
-<h3>Director Credits (<%= @person_movie_credits.directing.size + @person_tv_credits.directing.size %>)</h3>
-  <p>Movies (<%= @person_movie_credits.directing.size %>) </p>
+<h3>Director Credits (<%= movie_credits.directing.size + tv_credits.directing.size %>)</h3>
```

Comment:
This file is an example of the trend you'll see in all of the partials: replace the instance variables with partial locals